### DISCUSSION TITLE
In Frigate v0.14, where can I view the labeled images like I did in v0.13? I can only find short clips under the 'Review' menu

### DISCUSSION THREAD

**Discussion by mitrejit** [blakeblackshear/frigate#13344] [General]
In Frigate v0.14, where can I view the labeled images like I did in v0.13? I can only find short clips under the 'Review' menu

**Comment by NickM-27** [comment]
In 0.14 if frigate+ is enabled they can be viewed there, also in HA the media viewer still shows them. 

In frigate 0.15 there will be a new panel that shows all snapshots (if enabled) for the objects pertaining to a review item as well as other information like scores, zones, etc

**Comment by hawkeye217** [comment]
In addition to what @NickM-27 said, you should check out the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 8.
